program/calculate-compound-interest/CalculateCompoundInterest.js

```diff
@@ -0,0 +1,7 @@
+function calculateCompoundInterest(principal, rate, time, compoundingsPerYear) {
+  const compoundFactor = 1 + rate / 100 / compoundingsPerYear;
+  const numberOfCompoundings = time * compoundingsPerYear;
+  const compoundInterest =
+    principal * Math.pow(compoundFactor, numberOfCompoundings);
+  return compoundInterest - principal;
+}
```

Comment:
@ShivamK2002 can you please invoke the function with the values given in the issue .

Reply:
sure